cmd/retester/retester.go

```diff
@@ -180,12 +238,34 @@ const (
 	retestBackoffHold = iota
 	retestBackoffPause
 	retestBackoffRetest
-
-	maxRetestsForShaAndBase = 3
-	maxRetestsForSha        = 3 * maxRetestsForShaAndBase
+	maxRetestsForShaAndBaseDefault = 3
+	maxRetestsForShaDefault        = 3 * maxRetestsForShaAndBaseDefault
 )
 
-func (b *backoffCache) check(pr tide.PullRequest, baseSha string) (retestBackoffAction, string) {
+// Getter for max retests from config
+func (i *Info) getMaxRetests(pr tide.PullRequest) (int, int) {
```

Comment:
I would vote to have more descriptive return type even a struct... two ints don't say much.